editors/sc-ide/widgets/post_window.cpp

```diff
@@ -59,6 +59,8 @@ PostWindow::PostWindow(QWidget* parent): QPlainTextEdit(parent) {
     applySettings(Main::settings());
 }
 
+// Use QT QAction to setup post window functionalities
+// PostDocklet() is responsible for creating associated widgets if needed
```

Comment:
The first comment doesn't make sense.

The second would be better over in the header file.